.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "airbnb",
```

Comment:
Yeah, that one throws some people. I'll switch that rule around to enforce no dangling commas tomorrow morning.